tasks/create_target_vm/03_hosted_engine_final_tasks.yml

```diff
@@ -352,6 +352,11 @@
         register: vmremove_result
       - debug: var=vmremove_result
     when: local_vm_f.ansible_facts.ovirt_vms|length > 0
+  - name: Remove ovirt-engine-appliance rpm
+    yum:
+      name: ovirt-engine-appliance
+      state: absent
+    when: he_remove_appliance_rpm is true
```

Comment:
I think you can remove the "is true"
when: he_remove_appliance_rpm 
will be enough

Reply:
done